core/backend/src/annotations/TextBlockLayout.ts

```diff
@@ -401,6 +412,12 @@ export class RunLayout {
     });
   }
 
+  public applyTabShift(parent: LineLayout, context: LayoutContext) {
```

Comment:
```suggestion
  /** @internal */
  public applyTabShift(parent: LineLayout, context: LayoutContext) {
```

`LayoutContext` is not exported, no one outside this source file can use it so it doesn't belong in a public method. (I don't know why api-extractor didn't complain about that).

Reply:
Moved it to file scope.